### ISSUE TITLE
Whitespace inconsistency compared to pkg-config

### ISSUE THREAD

**Issue by jas4711** [pkgconf/pkgconf#278]
Hi.  We got a bug report that our regression tests for libgssglue failed, and it was tracked down to a difference between pkg-config and pkgconf.  Here is how to reproduce the difference:

```
jas@latte:~$ printf '%s\n' 'Name: foobar' 'Version: 1' 'Description: foobar' 'Cflags: -Ifoo' > foobar.pc 
jas@latte:~$ echo XXX`PKG_CONFIG_PATH=. pkgconf foobar --cflags`XXX
XXX-Ifoo XXX
jas@latte:~$ echo XXX`PKG_CONFIG_PATH=. pkg-config foobar --cflags`XXX
XXX-IfooXXX
jas@latte:~$ 
```

Notice the whitespace difference at end of line.  It doesn't come from the *.pc file, so it is introduced by pkgconf.  Would you consider fixing this minimal problem, to gain better compatibility with pkg-config?  Thank you!

**Comment by kaniini** [comment]
Fixed in 648a224.

**Comment by jas4711** [comment]
Thank you!
